src/api/MetadataQuery.js

```diff
@@ -133,12 +133,19 @@ class MetadataQuery extends Base {
      */
     flattenResponseEntry = ({ item, metadata }: MetadataQueryResponseEntry): FlattenedMetadataQueryResponseEntry => {
         const { id, name, size } = item;
+        /*
+            - In Content Explorer element, if can_preview permission is false, there is no action taken onClick(item).
+            - /metadata_queries api doesn't yet have permissions returned for each item in the response.
+            - Till the response has permissions, add can_preview: true so that users can click to launch the Preview modal. If users don't have access, they will see the error when preview loads.
```

Comment:
Minor:
- "Till" should be "Until"
- "Preview" is capitalized in the first sentence, but not in the second
- I also recommend placing quotes around "can_preview: true" to visually separate it from the rest of the sentence